source/about/people/volunteer_moderators.md

```diff
@@ -0,0 +1,30 @@
+# Volunteer Moderators
+
+(Content moderation)[/help/moderation/index.html] ensures that arXiv is a valuable resource for researchers. Submissions are checked that they appear in relevant subject areas and are of likely interest to professional researchers. arXiv relies on hundreds of volunteers, who are subject matter experts, to achieve its mission.
```

Comment:
Second sentence could be more direct and clear:
"arXiv moderators check that submissions appear in relevant..."